.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,53 @@
+# CI stages to execute against Pull Requests
+name: Robottelo - CI
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+
+env:
+    PYCURL_SSL_LIBRARY: openssl
```

Comment:
This customization shouldn't be necessary, I'd only add it if the install fails (see below comment)

Reply:
I left it in just in case, if that's ok.